packages/manager/apps/vrack-services/src/pages/RootWrapper.tsx

```diff
@@ -18,7 +19,7 @@ export default function RootWrapper() {
   React.useEffect(() => {
     trackCurrentPage();
 
-    const currentPage = (matches[matches.length - 1]?.handle as any)
+    const currentPage = (matches[matches.length - 1]?.handle as RouteConfig)
```

Comment:
handle is not a RouteConfig
It contains tracking and currentPage
```
type Handle = {
  tracking?: { pageType: PageType; pageName: PageName };
  currentPage?: string;
};
```

Reply:
tracking is not merge yet

Reply:
tracking on generator ?